linkup-cli/src/commands/deploy/resources.rs

```diff
@@ -904,6 +906,104 @@ impl TargetCfResources {
             notifier.notify("Linkup account token removed.");
         }
 
+        // Cleanup all the tunnels
+        let req = cloudflare::endpoints::cfd_tunnel::list_tunnels::ListTunnels {
+            account_identifier: &self.account_id,
+            params: cloudflare::endpoints::cfd_tunnel::list_tunnels::Params {
+                is_deleted: Some(false),
+                include_prefix: Some("linkup-tunnel-".to_string()),
```

Comment:
Maybe this prefix should similarly be like `linkup-tunnel-tunnel-zone-name`, would make it easier to delete and create the right things!

Reply:
What do you think of [a73a0d4](https://github.com/mentimeter/linkup/pull/195/commits/a73a0d4f35e1b9531dacfe8645435ce147a0da2b)?